app/konnect/getting-started/index.md

```diff
@@ -10,22 +10,22 @@ New to {{site.konnect_saas}}? Get started with the basics through the web app:
 
     [Create an account &gt;](/konnect/getting-started/access-account/)
 
-1.  **Set up a runtime instance**:
+1.  **Set up a data plane node**:
 
-    Set up your first runtime instance, which gives you a data plane for proxying traffic.
+    Set up your first data plane node, which gives you a data plane for proxying traffic.
 
     Start with a data plane proxy so that when your service configuration is
     ready, your services are immediately connected, configured,
     and available for testing.
```

Comment:
```suggestion

```
This doesn't really make sense to me ( I know you didn't write it). But I think this should be rewritten to talk about automatically connecting your data planes using the script in the Gateway manager.